message ObstacleMessage{
	required int32 direction = 1 [default = -1];
	required float distance = 2 [default = 0.0 ];
	required float certainty = 3 [default = 0.0];
	
}

message ObstacleMessageArray{
	repeated int32 direction = 1 ;
	repeated float distance = 2 ;
	repeated float certainty = 3 ;
	
}

message VisionObstacleMessage{
	repeated ObstacleMessage obstacles =1 ;	
}


message PathPlanningRequestMessage{
	required float goToX = 1 [default = 0.0];
	required float goToY = 2 [default = 0.0];
	required float goToAngle = 3 [default = 0.0];
}

message PathPlanningResultMessage{
	required float moveToX = 1 [default = 0.0];
	required float moveToY = 2 [default = 0.0];
	required float moveToAngle = 3 [default = 0.0];
}

message GridInfo{
	repeated float GridCells = 1 ; 
	repeated float OdometryMeasure = 2;
}
